README.md

```diff
@@ -71,7 +71,7 @@ Now installing the list of dependencies should no longer throw an error.
 
 #### Windows and OSX
 
-Tribler runs on Windows and OSX, but development is only supported on Linux.
```

Comment:
Sure, go ahead. 
Move the linux stuff that we already have on the README.md there too.